Tools/ci-run.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/bash
 
+set -x
+
```

Comment:
Echoing the commands makes it more clear what exactly is going on in the CI logs.